pkg/operator/starter.go

```diff
@@ -220,7 +221,29 @@ func WithSyncerImageHook() csidrivercontrollerservicecontroller.DeploymentHookFu
 	}
 }
 
-func newEnvVar(secretName, envVarName, key string) v1.EnvVar {
+// WithLogLevelDeploymentHook sets the X_CSI_DEBUG environment variable to a positive
+// value when CR.LogLevel is Debug or higher.
+func WithLogLevelDeploymentHook() csidrivercontrollerservicecontroller.DeploymentHookFunc {
```

Comment:
I don't see it used anywhere. In addition, should there be WithLogLevelDaemonSetHook?

Reply:
Sorry, I forgot to mark the PR as WIP

Reply:
Fixed